src/actions/identity-actions.ts

```diff
@@ -18,9 +18,20 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import {createAction} from 'redux-actions';
+import {createAction} from '@reduxjs/toolkit';
```

Comment:
Changing redux libraries seems like a big step, especially hidden in a big PR. But maybe this was pre-agreed?

Reply:
same ask here, is it for better typescript support?